CPAC/seg_preproc/seg_preproc.py

```diff
@@ -1072,6 +1072,110 @@ def create_seg_preproc_antsJointLabel_method(wf_name='seg_preproc_templated_base
     return preproc
 
 
+def create_seg_preproc_freesurfer(config=None, wf_name='seg_preproc_freesurfer'):
+
+    """
+    Generate the subject's segmentations based on freesurfer.
+
+    Parameters
+    ----------
+    wf_name : string
+        name of the workflow
+
+    Returns
+    -------
+    seg_preproc_freesurfer : workflow
+        workflow object for segmentation workflow
+
+    Notes
+    -----
+
+    Workflow Inputs: ::
+
+        inputspec.subject_dir : string (existing nifti file)
+            FreeSurfer autorecon1 dir
+
+    Workflow Outputs: ::
+
+        outputspec.wm_mask : string (nifti file)
+            outputs White Matter mask
+    """
+
+    preproc = pe.Workflow(name = wf_name)
+
+    inputnode = pe.Node(util.IdentityInterface(fields=['subject_dir']),
+                        name='inputspec')
+
+    outputnode = pe.Node(util.IdentityInterface(fields=['wm_mask',
+                                                        'gm_mask',
+                                                        'csf_mask',
+                                                        'subject_id']),
+                        name='outputspec')
+
+    reconall2 = pe.Node(interface=freesurfer.ReconAll(),
+                    name='anat_autorecon2')
+
+    reconall2.inputs.directive = 'autorecon2'
+    reconall2.inputs.openmp = config.num_omp_threads
+
+    if config.autorecon2_args is not None:
+        reconall2.inputs.args = config.autorecon2_args
```

Comment:
Good catch! These keys were used in autorecon1/2/3 implementation but since we use recon_all -all to fix concurrent issue, we can delete them for now. I just deleted these keys.